src/components/EcosystemInvite/SentInvitations.tsx

```diff
@@ -0,0 +1,257 @@
+	const [message, setMessage] = useState<string | null>(null);
+	const [error, setError] = useState<string | null>(null);
+	const [currentPage, setCurrentPage] = useState(initialPageState);
+  const [searchText, setSearchText]=useState<string>('')
+
+	const timestamp = Date.now();
+	const onPageChange = (page: number) => {
+		setCurrentPage({
+			...currentPage,
+			pageNumber: page,
+		});
+	};
+
+	const [invitationsList, setInvitationsList] = useState<Array<Invitation> | null>(null);
+	const props = { openModal, setOpenModal };
+
+	const getAllSentInvitations = async () => {
+		setLoading(true);
+		const response = await getEcosystemInvitations(
+			currentPage.pageNumber,
+			currentPage.pageSize,
+			searchText
+		);
+		const { data } = response as AxiosResponse;
+
+		if (data?.statusCode === apiStatusCodes.API_STATUS_SUCCESS) {
+			const totalPages = data?.data?.totalPages;
+
+			const invitationList = data?.data?.invitations;
+
+			setInvitationsList(invitationList);
+			setCurrentPage({
+				...currentPage,
+				total: totalPages,
+			});
+		} else {
+			setError(response as string);
+		}
+
+		setLoading(false);
+	};
+
+	useEffect(() => {
+		let getData: NodeJS.Timeout;
+
+		if (searchText.length >= 1) {
+			getData = setTimeout(() => {
+				getAllSentInvitations();
+			}, 1000);
+		} else {
+			getAllSentInvitations();
+		}
+		return () => clearTimeout(getData);
+	}, [searchText, openModal, currentPage.pageNumber]);
+
+	const searchInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+		setSearchText(e.target.value);
+	};
+
+	return (
+		<div className="px-4 pt-6">
+			<div className="mb-4 col-span-full xl:mb-2">
+				<BreadCrumbs />
+				
+			</div>
+			<div>
+				<div className="p-4 bg-white border border-gray-200 rounded-lg shadow-sm 2xl:col-span-2 dark:border-gray-700 sm:p-6 dark:bg-gray-800">
+					<h1 className="text-xl mb-4 font-semibold text-gray-900 sm:text-2xl dark:text-white">
+					Sent Ecosystem Invitations
+				</h1>
+
+					<AlertComponent
+						message={message ? message : error}
+						type={message ? 'success' : 'failure'}
+						onAlertClose={() => {
+							setMessage(null);
+							setError(null);
+						}}
+					/>
+
+					{loading ? (
+						<div className="flex items-center justify-center mb-4">
+							<CustomSpinner />
+						</div>
+					) : invitationsList && invitationsList?.length > 0 ? (
+						<div className="p-2 mb-4 bg-white border border-gray-200 rounded-lg shadow-sm 2xl:col-span-2 dark:border-gray-700 sm:p-3 dark:bg-gray-800">
+							<div className="flow-root">
+								<ul className="divide-y divide-gray-200 dark:divide-gray-700">
+									{invitationsList.map((invitation) => (
+										<li className="p-4">
+											<div className="flex flex-wrap justify-between xl:block 2xl:flex align-center 2xl:space-x-4">
+												<div className=" xl:mb-4 2xl:mb-0">
+													<div className="flex space-x-4">
+														<svg
+															width="60"
```

Comment:
Please format the file so it will look better

Reply:
Formatted.